database/src/lib.rs

```diff
@@ -45,3 +45,18 @@ impl<T> SqlxResultExt<T> for Result<T, sqlx::Error> {
         }
     }
 }
+
+#[cfg(test)]
+mod test_util {
+    use once_cell::sync::OnceCell;
+    use sqlx::PgPool;
+
+    pub async fn pool() -> &'static PgPool {
+        static INITIALIZED: OnceCell<()> = OnceCell::new();
+        if INITIALIZED.set(()).is_ok() {
+            dotenv::dotenv().unwrap();
+            migrations::initialize().await;
+        }
```

Comment:
What is this? :open_mouth:

Reply:
So, I don't know of a good way to initialize a once cell in an async environment. This is me enforcing some initialization to happen because tests don't have any way to have code run before any test is executed -- so this initialization code has to go somewhere.